managed/ui/src/components/config/Storage/ConfigType.js

```diff
@@ -0,0 +1,77 @@
+// Copyright (c) YugaByte, Inc.
+//
+// Author: Nishant Sharma(nishant.sharma@hashedin.com)
+//
+// This file will help us to maintain the different type of configs
+// along with their respective input field component except for AWS.
+
+import { YBTextInputWithLabel } from "../../common/forms/fields";
+
+const YBTextInputWithLabelComponent = YBTextInputWithLabel;
+const StorageConfigTypes = {
```

Comment:
given this is not a class or component - its name shouldn't start from a capital letter
```suggestion
export const storageConfigTypes = {
```

Reply:
Done